src/server/db/index.ts

```diff
@@ -0,0 +1,8 @@
+import Database from 'better-sqlite3';
+import { drizzle } from 'drizzle-orm/better-sqlite3';
+
+import * as schema from './schema';
+
+const sqlite = new Database('sqlite.db');
```

Comment:
Can this be made configurable? Also, default should be /app/sqlite.db

Reply:
Is now using env variable